src/assets/Generator.Shared/RequestUriBuilderExtensions.cs

```diff
@@ -6,120 +6,137 @@
 using System;
 using System.Collections.Generic;
 using System.Globalization;
+using System.Linq;
 using System.Xml;
 
 namespace Azure.Core
 {
     internal static class RequestUriBuilderExtensions
     {
+        private static string ConvertToString(object value, string? format = null)
```

Comment:
If all these extension methods go through the same code path, what is the value of having the separate extension methods?  Would it make sense to remove them and add a single `AppendPath(object value, string? format, bool escape)` method?  Why or why not?

Reply:
I did not do this change because I would like to minimize the change - this method is private therefore if we change the AppendPath to apply on `object`, we might have new cases get into this path, but if we do in this way, we will never have extra cases.

Reply:
Sounds good.